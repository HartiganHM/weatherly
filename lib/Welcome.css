.Welcome {
	/*display: absolute;*/
  border: 2px solid black;
  padding: 20px;
  margin: auto;
  height: 500px;
  width: 1000px;
}
