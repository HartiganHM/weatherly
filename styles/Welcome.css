.Welcome {
  color: ghostwhite;
  margin: auto;
  padding: 2rem;
  padding-top: 10rem;
}

.title {
  font-family: 'Pacifico', cursive;
  font-size: 7.2rem;
  text-align: center;
}
