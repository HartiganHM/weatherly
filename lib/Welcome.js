import React from "react";
import "../styles/Welcome.css";
import Search from "./Search.js";
import PropTypes from "prop-types";


const Welcome = props => {
  let greeting = true;
  return (
    <div className="Welcome">
      <p className="title">Weatherly</p>
      <Search fetch={props.fetch} greeting={greeting} />
    </div>
  );
};

Welcome.propTypes = {
  fetch: PropTypes.func
};

export default Welcome;
