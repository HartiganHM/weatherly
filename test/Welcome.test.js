import React from 'react';
import { shallow, mount } from 'enzyme';
import Welcome from '../lib/Welcome';

describe('Welcome', () => {

    it('Welcome should exist', () => {
        expect(<Welcome />).toBeDefined()
    })

    it('Should have a className Welcome', () => {
      const welcome = shallow(<div className='Welcome'></div>);
        expect( welcome.find('.Welcome').length ).toEqual(1);
    })
});