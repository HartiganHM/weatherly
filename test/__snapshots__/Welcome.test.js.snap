// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome Should have a className Welcome 1`] = `[Function]`;
